nl.emphisia.icon.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id" : "nl.emphisia.icon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "runtime-version" : "46",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "folder_icon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag": "v0.8.1"
+                }
+            ]
+        },
+        {
+            "name" : "folder_icon",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://github.com/youpie/Iconic/releases/download/Major/Iconic-Release.tar.xz",
```

Comment:
this doesn't look like a versioned release.

Reply:
ah, you mean like specifically version 1.0.0, this is just the link i got when i copied the url from the release :)